### PR TITLE
Created the event detail footer

### PR DESCRIPTION
**Pull request by jimmynotjim**
Created the event detail footer

## Additions

- Created event footer banner for RSVP message with reservation button
- Created PDF download link
- Created questions email link
- Added topdoc for event detail footer

## Removals

- None

## Changes

- None

## Testing

- navigate to `/events/spring-2014-rulemaking-agenda/`

## Review

- @sebworks 
- @anselmbradford 
- @duelj 

## Preview

screen shot 2015-02-24 at 5 37 48 pm

screen shot 2015-02-24 at 5 37 59 pm

## Notes

- Updated the button and link styles to match specifications in design manual

### CODE REVIEW

File: events/_single.html [hunk 1]
```diff
@@ -24,6 +24,23 @@
 {% block content_main %}
     {% import "_event.html" as article with context %}
     {{ article.render(vars.mock_event, vars.path) }}
+
+    <div class="event-footer">
+        <p class="event-footer_banner">
+            <span class="event-footer_banner_message">This event requires an RSVP. Reserve your spot here: </span>
```
Comment (anselmbradford) [inline events/_single.html]: `event-footer_banner_message` or `event-footer_banner-message`?

File: static/css/event.less [hunk 2]
```diff
@@ -13,3 +13,105 @@
...
+                </section>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          div.event-footer
+            p.event-footer_banner
+              span.event-footer_message
```
Comment (anselmbradford) [inline static/css/event.less]: `event-footer_message` is not used.

File: static/css/event.less [hunk 3]
```diff
@@ -13,3 +13,105 @@
...
+        }
+    });
+
+    .respond-to-min(600px, {
+        &_col:nth-child(odd):not(:last-child) {
+            border-right: 1px solid #ddd;
+        }
+    });
+
+    &_banner {
```
Comment (anselmbradford) [inline static/css/event.less]: I'd lean toward avoiding building class names like this, per our discussion about them being hard to find, unless we have a plan to include a sourcemap that will help with this. It'll reduce bracket nesting too if we directly include the full class name.

File: events/_single.html (hunk 1 above)
Comment (anselmbradford) [inline events/_single.html]: Not sure which is BEM correct here, but looking at https://github.com/cfpb/capital-framework/issues/40#issuecomment-36789156

Comment (jimmynotjim) [inline events/_single.html]: Yeah, I think I overdid it here. Adjusting.

File: static/css/event.less (hunk 2 above)
Comment (jimmynotjim) [inline static/css/event.less]: Good catch. I kept changing these and forgot to update the topdoc.

File: static/css/event.less (hunk 3 above)
Comment (jimmynotjim) [inline static/css/event.less]: I'm in favor of keeping with the existing class pattern and fixing the source maps, something I was going to look at this week. I also don't feel bracket nesting is an issue here due to specificity not being affected. Rules on limiting nesting are really more about specificity than code organization.

**Comment by jimmynotjim** [comment]
Updated with design requests from @duelj and ready for sign-off.

**Comment by sebworks** [comment]
:+1:

**Comment by anselmbradford** [comment]
Looks good. There's an alignment issue we might try to tackle at some point on the RSVP buttons, but that varies with screensize:

screen shot 2015-02-25 at 9 30 42 am

**Comment by jimmynotjim** [comment]
Good point. I tried to quickly fix it using the CF grid but it didn't work due to styles added to the container. Do you want to take a look at this while I finish up the agenda table? 

I did update the post-inset button to include the icon left styles and pushed to sprint-21

**Pull request merged**